src/main/java/com/botdarr/api/sonarr/SonarrCommands.java

```diff
@@ -15,19 +15,17 @@ public static List<Command> getCommands(SonarrApi sonarrApi) {
     return new ArrayList<Command>() {{
       add(new BaseCommand(
               "show id add",
-        "Adds a show using search text and tmdb id (i.e., show id add 30 clock 484767)",
-              Arrays.asList("show-title", "show-tvdbid")) {
+        "Adds a show using search text and tmdb id (i.e., show id add 484767)",
```

Comment:
im okay with this breaking change, however you'd also need to change

SonarrCommands#getAddShowCommandStr

Basically we append the show id <show-title> <show-id> into the responses for each chat client (see https://github.com/shayaantx/botdarr/blob/0ea239a146be667c962c5d5120634a13dcc592cf/src/main/java/com/botdarr/clients/slack/SlackResponseBuilder.java#L351 for example)

We do this by default so that regardless of chat client a user of the bot, can just copy and paste the exact command to add a show.

Now thats the minimum, other clients like discord support thumbs for action for adding or slash commands, slack also supports thumbs up, telegram supports interactive commands with buttons, etc

So with that being said, you could change this api or add another one technically like "show id-only add" or whatever name you can come up with?

thoughts?

Reply:
Please give a look at the last changes when you can. I believe this is the change you're mentioning, right?
![image](https://github.com/shayaantx/botdarr/assets/27465604/8cd3e74d-3ff3-47a7-90ab-fd446c98ef1c)

Reply:
Although I didn't test for the other chat clients. I'm going to try to do that.

Reply:
looks good to me, let me know how some of the other tests go, also if you can bump the version in version file (https://github.com/shayaantx/botdarr/blob/development/src/main/resources/version.txt) that should be everything (sorry the version management is kinda whack)